usaspending_api/etl/tests/integration/test_load_transactions_in_delta.py

```diff
@@ -0,0 +1,1345 @@
+        unique_award_key="award_procure_0001"
+    )
+    baker.make(
+        "awards.TransactionNormalized",
+        id=3,
+        award=assist_awards[1],
+        transaction_unique_id="award_assist_0002_trans_0001",
+        update_date=datetime(year=2022, month=10, day=31),
+        is_fpds=False,
+        unique_award_key="award_assist_0002"
+    )
+    baker.make(
+        "awards.TransactionNormalized",
+        id=4,
+        award=procure_awards[1],
+        transaction_unique_id="award_procure_0002_trans_0001",
+        update_date=datetime(year=2022, month=10, day=31),
+        is_fpds=True,
+        unique_award_key="award_procure_0002"
+    )
+    baker.make(
+        "awards.TransactionNormalized",
+        id=5,
+        award=assist_awards[1],
+        transaction_unique_id="award_assist_0002_trans_0002",
+        update_date=datetime(year=2022, month=10, day=31),
+        is_fpds=False,
+        unique_award_key="award_assist_0002"
+    )
+    baker.make(
+        "awards.TransactionNormalized",
+        id=6,
+        award=procure_awards[1],
+        transaction_unique_id="award_procure_0002_trans_0002",
+        update_date=datetime(year=2022, month=10, day=31),
+        is_fpds=True,
+        unique_award_key="award_procure_0002"
+    )
+    baker.make(
+        "awards.TransactionNormalized",
+        id=7,
+        award=assist_awards[2],
+        transaction_unique_id="award_assist_0003_trans_0001",
+        update_date=datetime(year=2022, month=10, day=31),
+        is_fpds=False,
+        unique_award_key="award_assist_0003"
+    )
+    baker.make(
+        "awards.TransactionNormalized",
+        id=8,
+        award=assist_awards[2],
+        transaction_unique_id="award_assist_0003_trans_0002",
+        update_date=datetime(year=2022, month=10, day=31),
+        is_fpds=False,
+        unique_award_key="award_assist_0003"
+    )
+    baker.make(
+        "awards.TransactionNormalized",
+        id=9,
+        award=procure_awards[2],
+        transaction_unique_id="award_procure_0003_trans_0001",
+        update_date=datetime(year=2022, month=10, day=31),
+        is_fpds=True,
+        unique_award_key="award_procure_0003"
+    )
+    baker.make(
+        "awards.TransactionNormalized",
+        id=10,
+        award=procure_awards[2],
+        transaction_unique_id="award_procure_0003_trans_0002",
+        update_date=datetime(year=2022, month=10, day=31),
+        is_fpds=True,
+        unique_award_key="award_procure_0003"
+    )
+
+    # `name` and `external_data_type_id` must match those in `usaspending.broker.lookups`
+    edt_tn = baker.make(
+        "broker.ExternalDataType",
+        name="transaction_normalized",
+        external_data_type_id=203,
+        update_date="2022-10-31"
+    )
+    baker.make("broker.ExternalDataLoadDate", last_load_date="2022-10-31", external_data_type=edt_tn)
+
+@fixture
+def finish_external_data_tables(populate_initial_awards_and_transaction_normalized_data):
```

Comment:
I see that this fixture relies on `populate_initial_awards_and_transaction_normalized_data`, which in turn relies on both  `populate_initial_source_assistance_transaction_data` and `populate_initial_source_procurement_transaction_data`. If all fixtures are being used by all tests, why have them divided out into separate fixtures?

Reply:
Done

Reply:
Nice job keeping them organized with comments 👍